scripts/prerequisites.sh

```diff
@@ -0,0 +1,10 @@
+S3_BUCKET_SRC=trady-cloud-src
+S3_BUCKET_TERRAFORM=trady-cloud-terraform
+REGION=us-west-2
+
+# Create buckets necessary for Terraform via script as the sandbox does not allow creating them via Terraform itself
+sh setup_buckets.sh $REGION $S3_BUCKET_SRC $S3_BUCKET_TERRAFORM 
+
+# Create from terraform-templates with variables: main.tf & variables.tf
+sed -e s/REGION/${REGION}/ -e s/S3_BUCKET_TERRAFORM/${S3_BUCKET_TERRAFORM}/  < ../infrastructure/templates/main.tf.template > ../infrastructure/main.tf
+sed s/REGION/${REGION}/ < ../infrastructure/templates/variables.tf.template > ../infrastructure/variables.tf
```

Comment:
do you really need this? 
You can also parse this parameters to terraform as configuration from your environment variables

Reply:
I thought about maybe let the user later decide this via terminal input and wrote it like this to be as flexible as possible

Reply:
but you can do this with running 

`terraform init -input=false -backend-config="bucket=$TERRAFORM_STATE_BUCKET"`

Than you do not need to replace some thing

Also it is always better to define some thing when you do not need to configure it now

"convention over configuration" 

Helps you to reduce complexity in your code

Reply:
Whaaaaat? Are you trying to reduce my second name?!?! 🧐

Reply:
See following suggestions